client/lib/user/support-user-interop.js

```diff
@@ -78,7 +77,7 @@ const _isSupportUserSession = ( () => {
 		return false;
 	}
 
-	const supportUser = store.get( STORAGE_KEY );
+	const supportUser = JSON.parse( sessionStorage.getItem( STORAGE_KEY ) );
```

Comment:
JSON.parse might throw. Let's add a try catch, where we return false in the catch.

Reply:
updated on 04e33a972fe2a8530051d8d2c12b13850acca81b.